internal/driver/driver_test.go

```diff
@@ -796,6 +799,166 @@ func heapProfile() *profile.Profile {
 	}
 }
 
+func heapProfileMedium() *profile.Profile {
```

Comment:
I switched to making the profile used to test from heapProfile(), and now call it "heap_sizetags"